tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -41,7 +41,7 @@ describe('Performing an upgrade', () => {
     await (await upgradePage.deploymentComplete()).waitForDisplayed();
 
     const currentVersion = await upgradePage.getCurrentVersion();
-    expect(currentVersion).to.include(version.getVersion());
+    expect(version.getVersion(true)).to.include(currentVersion);
```

Comment:
Can this just be `equal` instead of `include` since `getVersion` should not be returning the build number now?

Reply:
`getVersion` _is_ including the build number. `currentVersion` isn't. It'd take a little bit of tinkering, and right now anyone's build will be broken. :(